aiida_pseudo/cli/install.py

```diff
@@ -140,3 +140,111 @@ def cmd_install_sssp(version, functional, protocol, traceback):
+    from aiida_pseudo import __version__
+    from aiida_pseudo.groups.family import (
+        PseudoDojoConfiguration, PseudoDojoFamily, PseudoDojoPsmlFamily, PseudoDojoPsp8Family, PseudoDojoUpfFamily
+    )
+
+    from .utils import attempt, create_family_from_archive
+
+    pseudo_format_families = {'psp8': PseudoDojoPsp8Family, 'psml': PseudoDojoPsmlFamily, 'upf': PseudoDojoUpfFamily}
+
+    metadata_urls = {
+        'nc-sr-04_pbe_standard': 'ONCVPSP-PBE-PDv0.4/standard.djson',
+        'nc-sr-04_pbe_stringent': 'ONCVPSP-PBE-PDv0.4/stringent.djson',
+        'nc-sr-04_pbesol_standard': 'ONCVPSP-PBEsol-PDv0.4/standard.djson',
+        'nc-sr-04_pbesol_stringent': 'ONCVPSP-PBEsol-PDv0.4/stringent.djson',
+        # 'nc-fr-04_pbe_standard': 'ONCVPSP-PBE-FR-PDv0.4/standard.djson',  # md5 does not match
+        # 'nc-fr-04_pbe_stringent': 'ONCVPSP-PBE-FR-PDv0.4/stringent.djson'  # missing elements
+    }
+
+    configuration = PseudoDojoConfiguration(version, functional, rel, protocol, hint, pseudo_format)
+    label = PseudoDojoFamily.format_configuration_label(configuration)
+    description = f'PseudoDojo v{version} {functional} {rel} {protocol} {hint} {pseudo_format} \
+        installed with aiida-pseudo v{__version__}'
+
+    if pseudo_format not in pseudo_format_families:
+        echo.echo_critical(f'{pseudo_format} is not a valid PseudoDojo pseudopotential format')
+    else:
+        pseudo_dojo_format_family = pseudo_format_families[pseudo_format]
+
+    if configuration not in pseudo_dojo_format_family.valid_configurations:
+        echo.echo_critical(f'{version} {functional} {protocol} is not a valid PseudoDojo configuration')
+
+    if QueryBuilder().append(pseudo_dojo_format_family, filters={'label': label}).first():
+        echo.echo_critical(f'{pseudo_dojo_format_family.__name__}<{label}> is already installed')
+
+    with tempfile.TemporaryDirectory() as dirpath:
+
+        if rel == 'sr3plus':
+            url_archive = f'{URL_PSEUDODOJO_BASE}/nc-sr-{version}-3plus_{functional}_{protocol}_{pseudo_format}.tgz'
+        else:
+            url_archive = f'{URL_PSEUDODOJO_BASE}/nc-{rel}-{version}_{functional}_{protocol}_{pseudo_format}.tgz'
+        filepath_archive = os.path.join(dirpath, 'archive.tgz')
+
+        with attempt('downloading selected pseudo potentials archive... ', include_traceback=traceback):
+            response = requests.get(url_archive)
+            response.raise_for_status()
+            with open(filepath_archive, 'wb') as handle:
+                handle.write(response.content)
+                handle.flush()
+                description += f'\nArchive pseudos md5: {md5_file(filepath_archive)}'
+
+        with attempt('unpacking archive and parsing pseudos... ', include_traceback=traceback):
+            family = create_family_from_archive(pseudo_dojo_format_family, label, filepath_archive)
+
+        family.description = description
+
+        if hint != 'none':
```

Comment:
Seeing this concept of the pseudo dojo that there are multiple recommended cutoffs for a family, makes me think whether we should not just include this into the code as a default, i.e., instead of creating multiple groups of a pseudo family, each one with a particular set of recommended cutoffs, maybe we create a single group and add *all* recommended cutoffs and set one of those as the default. I think this makes more sense. The SSSP can just provide the single set as it does now. What do you think? If you agree, I will first make that adaptation to the code in general and then you can just use it for this PR

Reply:
I think that would cut down on a lot of repetition in `aiida-pseudo`, and it means in the end, you don't end up with 50 different pseudo families. 

But it kind of kicks the problem down the road. This means that the user then has to pull this information out manually each time they use the pseudo family, and they have to remember what every pseudo family contains in terms of cutoffs (is it none, one set, many sets? what are the names that this specific family calls their different levels of cutoffs?). 

I'm not entirely certain where we want to deal with it. If we do it here, then the cutoffs are uniquely determined by the pseudopotential family from which the pseudos are retrieved, and the level of precision is clearly stated in the pseudo family label. If we do it down the line, then we risk confusing and frustrating users when they have to deal themselves with knowing what kind(s) of cutoff suggestions each family contains and how to properly specify their choice. 

I guess what I'm saying is that in my mind, it's almost a choice between outward cleanliness and ease of use, but that could totally be remedied with some smart implementation details.

Reply:
Fair point, but as I suggested we should of course include a default suggested cutoff set. So even with this change, doing `family.get_recommended_cutoffs(structure=structure)` will still work. However, it will also allow you to do `family.get_recommended_cutoffs(structure=structure, stringency='high')`. We can easily add a method that will provide the available stringencies: `family.get_available_cutoff_stringencies() == ['low', 'normal', 'high']` and `family.get_default_cutoff_stringency() == 'normal'`

Reply:
Please take a look at my implementation